packages/kotlin/src/generators/services/okhttp3-clients/okhttp3-client-generator.ts

```diff
@@ -67,7 +67,9 @@ export class DefaultKotlinOkHttp3Generator extends KotlinFileGenerator<Context,
       primaryConstructor: kt.constructor(
         [
           kt.parameter('basePath', kt.refs.string(), { default: 'defaultBasePath' }),
-          kt.parameter('client', kt.refs.okhttp3.okHttpClient(), { default: kt.call(['ApiClient', 'defaultClient']) }),
+          kt.parameter('client', kt.refs.okhttp3.okHttpClient(), {
+            default: kt.call(['defaultClient']),
```

Comment:
`kt.call` is unnecessary here. You can just write `default: 'defaultClient'`

Reply:
Done